src/containers/DataTable/index.js

```diff
@@ -15,7 +15,7 @@ import {
 
 import Table from './Table';
 import Pagination from './Pagination';
-import CustomDropdownButton from './CustomDropdownButton';
+// import CustomDropdownButton from './CustomDropdownButton'; DON'T REMOVE YET. IT MAY BE USEFUL ON REFACTORING DATATABLE
```

Comment:
I would write `//` also before `DON'T REMOVE YET` for easy uncommenting
`// import CustomDropdownButton from './CustomDropdownButton'; // DON'T REMOVE YET. IT MAY BE USEFUL ON REFACTORING DATATABLE`

Reply:
That's okay. I will be working on this probably next sprint and will delete this myself.